QUARKUS-1407.md

```diff
@@ -0,0 +1,38 @@
+# Quarkus-1407 - Testing and verifying Oracle SQL client
+
+The goal of this test plan is to cover the main points Oracle SQL client in Quarkus. 
```

Comment:
This sentence is a bit redundant and doesn't make much sense. Let's put some content in or omit it.

Reply:
I copypasted it, but edited only half of it >_<

Fixed